packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -338,11 +348,24 @@ abstract class DatabaseClusterNew extends DatabaseClusterBase {
     ];
 
     let { s3ImportRole, s3ExportRole } = setupS3ImportExport(this, props, /* combineRoles */ false);
+
+    if (props.parameterGroup && props.parameters) {
+      throw new Error('You cannot specify parameterGroup and parameters');
+    }
+    const parameterGroup = props.parameterGroup ?? (
+      props.parameters ?
+        new ParameterGroup(this, 'ParameterGroup', {
+          engine: props.engine,
+          description: 'Auto-generated parameter group',
```

Comment:
I would let the default of `ParameterGroup` act here.

```suggestion
```

Reply:
Yep, +1.